crates/paimon/src/spec/schema.rs

```diff
@@ -51,3 +51,133 @@ pub struct DataField {
     typ: DataType,
     description: Option<String>,
 }
+
+impl DataField {
+    pub fn new(id: i32, name: String, typ: DataType) -> Self {
+        Self {
+            id,
+            name,
+            typ,
+            description: None,
+        }
+    }
+
+    pub fn id(&self) -> i32 {
+        self.id
+    }
+
+    pub fn name(&self) -> &str {
+        &self.name
+    }
+
+    pub fn data_type(&self) -> &DataType {
+        &self.typ
+    }
+
+    pub fn description(&self) -> Option<&String> {
```

Comment:
Better to return `Option<&str>` instead. `&String` is rarely used in rust.

Reply:
This is my oversight, I will fix it and remember this usage convention

Reply:
done